ui/js/models/resultset.js

```diff
@@ -227,12 +227,56 @@ treeherder.factory(
              },
 
              triggerNewJobs: function(repoName, resultset_id, buildernames, decisionTaskID) {
-                 var uri = resultset_id + '/trigger_runnable_jobs/';
```

Comment:
There may already be one floating around that we can repurpose, but when this lands could you file a bug for removing this API and corresponding Treeherder parts? (I guess after that we can also file another for the pulse_actions side too)